osgp/shared/osgp-dto/src/main/java/org/opensmartgridplatform/dto/valueobjects/smartmetering/PowerQualityObjectDto.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright 2017 Smart Society Services B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.dto.valueobjects.smartmetering;
+
+import java.io.Serializable;
+
+public class PowerQualityObjectDto implements Serializable {
+
+    private static final long serialVersionUID = 2123390296585369208L;
+
+    private final String name;
+    private final String unit;
+
+    public PowerQualityObjectDto(final String name, final String unit) {
+        super();
```

Comment:
I have seen this in multiple places. Calling the no-arg super constructor is somehting the compiler inserts by default. Better not to put in the code explicitly.

Reply:
removed super(); Introduced @lombok.Getter and removed get methods.